ibis/spark/tests/test_udf.py

```diff
@@ -303,11 +303,15 @@ def my_mean(series):
         ['key', 'a']
     )
     result = expr.execute()
-    expected = df_random.sort_values(['key', 'a']).assign(
-        rolled=lambda df: df.groupby('key')
-        .b.rolling(3, min_periods=1)
-        .mean()
-        .reset_index(level=0, drop=True)
+    expected = (
+        df_random.sort_values(['key', 'a'])
+        .assign(
+            rolled=lambda df: df.groupby('key')
+            .b.rolling(3, min_periods=1)
+            .mean()
+            .reset_index(level=0, drop=True)
+        )
+        .reset_index(drop=True)
```

Comment:
This is an `xfail` test that should not fail if `pyspark` is `3.0.0`.

When I had `pyspark` at `3.0.0`, this test failed anyways because it was missing a `.reset_index(drop=True)`. I'm leaving this fix in (even though I've now made `pyspark<3.0.0`) so that when we un-`xfail` this test in the future, it will work.

Reply:
Sounds good